src/tests/functional/inference_engine/lib_close.cpp

```diff
@@ -0,0 +1,103 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "lib_close.hpp"
+
+#include "gtest/gtest.h"
+#include "openvino/frontend/manager.hpp"
+#include "openvino/runtime/core.hpp"
+#include "openvino/util/file_util.hpp"
+
+using namespace testing;
+using namespace ov::util;
+using namespace ov::frontend;
+
+std::string FrontendLibCloseTest::get_test_case_name(const testing::TestParamInfo<FrontendLibCloseParams>& obj) {
+    return std::get<0>(obj.param);
+}
+
+void FrontendLibCloseTest::SetUp() {
+    std::tie(frontend, model_path, exp_name) = GetParam();
+}
+
+/**
+ * \brief Delete InputModel object as last.
+ *
+ * Frontend library must close after object deletion, otherwise segfault can occur.
+ */
+TEST_P(FrontendLibCloseTest, testModelIsLasDeletedObject) {
+    InputModel::Ptr model;
+    {
+        auto fem = std::make_shared<ov::frontend::FrontEndManager>();
+        auto fe = fem->load_by_framework(frontend);
+        model = fe->load(model_path);
+    }
+    ASSERT_NE(model, nullptr);
+}
+
+/** \brief Frontend library must close after object deletion, otherwise segfault can occur. */
+TEST_P(FrontendLibCloseTest, testPlaceIsLastDeletedObject) {
+    Place::Ptr place;
+    {
+        auto fem = std::make_shared<ov::frontend::FrontEndManager>();
+        auto fe = fem->load_by_framework(frontend);
+        auto model = fe->load(model_path);
+        place = model->get_place_by_tensor_name(exp_name);
+    }
+
+    ov::shutdown();
+
+    ASSERT_NE(place, nullptr);
+    EXPECT_EQ(place->get_names().at(0), exp_name);
+}
+
+/** \brief Frontend library must close after object deletion, otherwise segfault can occur. */
+TEST_P(FrontendLibCloseTest, testPlaceIsLastDeletedObject2) {
+    EXPECT_DEATH(
+        {
+            Place::Ptr place;
+            {
+                auto fem = std::make_shared<ov::frontend::FrontEndManager>();
+                auto fe = fem->load_by_framework(frontend);
+                auto model = fe->load(model_path);
+                place = model->get_place_by_tensor_name(exp_name);
+            }
+
+            ov::shutdown();
+
+            ASSERT_NE(place, nullptr);
+            EXPECT_EQ(place->get_names().at(0), exp_name);
+        },
+        ".*");
+}
```

Comment:
Why do we have `EXPECT_DEATH` only for one test?

Reply:
This it additional test which should make segfault to check if ov::shutdown  close libs. 
Other check positive scenario if such think not occurs. I think one is enough.
But is should be renamed, it looks like I lose some local changes :(.